emails/views.py

```diff
@@ -389,6 +390,8 @@ def _get_all_contents(email_message):
         histogram_if_enabled(
             'attachment.count_per_email', len(attachments)
         )
+        if text_content and html_content is None:
```

Comment:
can we make this logic even more explicit for clarity?
```suggestion
        if text_content is not None and html_content is None:
```